lib/identity_cache.rb

```diff
@@ -56,15 +66,9 @@ class << self
     self.fetch_read_only_records = version >= Gem::Version.new("0.5")
 
     def included(base) #:nodoc:
```

Comment:
If we are using `concern` we should call `super` here or use `included` block instead of defining the method.